ppoprf/examples/server.rs

```diff
@@ -19,28 +19,32 @@
 //! curl --silent localhost:8080 \
 //!     --header 'Content-Type: application/json' \
 //!     --data '{"name":"Nested STAR", "points": [
-//!         [226, 242, 174, 10, 106, 188, 78, 113,
-//!          168, 132, 169, 97, 197, 0, 81, 95,
-//!          88, 227, 11, 106, 165, 130, 221, 141,
-//!          182, 166, 89, 69, 224, 141, 45, 118]]}'
+//!         "4vKuCmq8TnGohKlhxQBRX1jjC2qlgt2NtqZZReCNLXY="
+//!         ]}'
 //! ```
 
 use actix_web::middleware::Logger;
 use actix_web::{get, post, web};
 use env_logger::Env;
+use dotenv::dotenv;
+use std::env;
 use log::info;
 
 use std::sync::{Arc, RwLock};
 
-use curve25519_dalek::ristretto::CompressedRistretto;
 use ppoprf::ppoprf;
 
 use serde::{Deserialize, Serialize};
 
+const DEFAULT_EPOCH_DURATION: u64 = 5;
+const DEFAULT_MDS: &str = "116;117;118;119;120";
+const EPOCH_DURATION_ENV_KEY: &str = "EPOCH_DURATION";
```

Comment:
These keys are only used once. For a quick example like this, I think it's better to just use the literal directly at the single point of access.

Reply:
Or you could log something about the supported key names and their values if set. That would help discovery and be a reason to keep the `_ENV_KEY` strings around. :)